### ISSUE TITLE
Using numbers in suffix

### ISSUE THREAD

**Issue by jakjr** [jonathanpmartins/v-money3#61]
Hello, I am trying to active this result: "R$ 999,00"

The cents should be fixed, so this is my config:

`
      config: {
        prefix: 'R$ ',
        suffix: ",00",
        thousands: '.',
        decimal: '',
        precision: 0,
        disableNegative: true,
        disabled: false,
        min: null,
        max: null,
        allowBlank: true,
        minimumNumberOfCharacters: 0,
      }

`
The result is: "R$ 999,", omitting the both zeros in the suffix.

Using letters in suffix works like expected: "R$ 999,AA".

**Comment by jonathanpmartins** [comment]
Yep, that's really not supported right now. A pull request will be apreciated!
